types/hapi__call/index.d.ts

```diff
@@ -1,29 +1,33 @@
-// Type definitions for @hapi/call 8.0
+// Type definitions for @hapi/call 9.0.0
 // Project: https://github.com/hapijs/call#readme
 // Definitions by: Rodrigo Saboya <https://github.com/saboya>
+//                 Sebastian Malton <https://github.com/nokel81>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 
+export type RouteMethod = "get" | "head" | "post" | "put" | "delete" | "connect" | "options" | "trace" | "patch";
+
 export interface RouteDefinition {
-    method: string;
+    method: RouteMethod | "*";
```

Comment:
The documentation on the hapi website states:

> method - the HTTP method ('get', 'put', 'post', 'delete', etc.) or the wildcard character ('*') to match any methods. The method must be lowercase.

Reply:
Nothing (besides questionable taste) stops people from using non-standard HTTP verbs. This may be too prescriptive. Also, the implementation is specifically [designed to be case-insensitive](https://github.com/hapijs/call/blob/f4adca38ae29852fa5f990b13a0abbc36a53b924/lib/index.js#L37), so this is potentially breaking for no reason.

Reply:
Okay, will revert this.